server/plugin.go

```diff
@@ -116,6 +118,7 @@ func (p *Plugin) initializeAPI() {
 
 	p.router.HandleFunc("/add", p.checkAuth(p.handleAdd)).Methods(http.MethodPost)
 	p.router.HandleFunc("/list", p.checkAuth(p.handleList)).Methods(http.MethodGet)
```

Comment:
Do we need this endpoint anymore? Or are we still calling this in some cases?

Reply:
No need so i delete it but getIssueList is used by command